ui/src/components/app-form/app-form.tsx

```diff
@@ -433,321 +624,360 @@ export const AppForm = ({ id }: AppFormProps): React.ReactElement => {
+                      disabled={!isUrlValid} // Disable field until URL is validated
+                    />
+                  </FormControl>
+                )}
               />
-            </FormControl>
-          )}
-        />
-
-        <Controller
-          name="description"
-          control={control}
-          render={({ field }) => (
-            <FormControl
-              fullWidth
-              className="form-control outer-div"
-              variant="outlined"
-              style={{ position: 'relative' }}
-            >
-              <label
-                htmlFor="description"
-                className="description-label"
-                style={{
-                  position: 'absolute',
-                  top: '0',
-                  left: '10px',
-                  transform: 'translate(0, -50%)',
-                  color: isFocused ? '#ba18da' : '#646464',
-                  backgroundColor: '#fafafa',
-                  padding: '0 4px',
-                  zIndex: 1,
-                }}
-              >
-                Description
-              </label>
-              <textarea
-                {...field}
-                ref={textAreaRef}
-                id="description"
-                value={description}
-                onChange={(e) => {
-                  setDescription(e.target.value);
-                  field.onChange(e.target.value);
-                  adjustTextareaHeight(e.target);
-                }}
-                onFocus={() => handleFocus(true)}
-                onBlur={() => {
-                  field.onBlur();
-                  handleFocus(false);
-                }}
-                className="description_text-field"
-                placeholder="Add app description (max. 200 characters)"
-                style={{
-                  paddingBottom: '8px',
-                  borderColor: isFocused ? '#ba18da' : '#ccc',
-                  letterSpacing: '.15px',
-                  backgroundColor: '#FAFAFA',
-                }}
+              {/* Conda YAML Directory Input */}
+              <Controller
+                name="conda_project_yml"
+                control={control}
+                render={({ field }) => (
+                  <FormControl fullWidth>
+                    {/* Label for Conda YAML Directory */}
+                    <InputLabel
+                      htmlFor="conda_project_yml"
+                      shrink
+                      sx={labelStyle}
+                    >
+                      Conda YAML Directory
+                    </InputLabel>
+
+                    {/* Conda YAML Directory TextField */}
+                    <TextField
+                      {...field}
+                      id="conda_project_yml"
+                      placeholder="Enter path to conda-project.yml"
+                      disabled={!isCondaYamlEnabled} // Disable until configuration is fetched
```

Comment:
I don't see why we would need to disable this. The user would need to (optionally) supply this, before we fetch the app configuration.

Same applies for branch input.

Reply:
Also, I am unable to input anything in this field even after the app configuration is fetched.

Reply:
Adjusted, removed the disbled ocndition on conda yaml and branch inputfields, fixed input field to take input. 

Intent was to get the information first, then user could make adjustments.